src/versioning/steamcmd.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+from constants import APP_ID, DEPOT_ID
+
+
+class SteamCMD:
+    """
+    SteamCMD is a Valve tool that allows some steam operations from the command line
+
+    This class is used to retrieve the most recent manifest-id for Deadlock
+    """
+
+    def __init__(self, steam_cmd_path, steam_username, steam_password):
+        self.STEAM_CMD_PATH = steam_cmd_path
+        self.APP_ID = APP_ID
+        self.DEPOT_ID = DEPOT_ID
+        self.steam_username = steam_username
+        self.steam_password = steam_password
+
+    def run(self):
```

Comment:
Since the class is called `SteamCMD`, I'd call this function `get_latest_manifest_id`, which would make the comment in the class not needed. Leaves us room to use this class for other purposes than just that